repose-aggregator/components/filters/compression-filter/src/main/scala/org/openrepose/filters/compression/CompressionFilter.scala

```diff
@@ -91,6 +93,7 @@ class CompressionFilter @Inject()(configurationService: ConfigurationService, co
           logger.error("Error with the CompressingFilter", e)
           httpResponse.setStatus(HttpServletResponse.SC_INTERNAL_SERVER_ERROR)
       }
+      httpResponse.commitToResponse()
```

Comment:
It wasn't required since the powerfilter would eventually commit the stream and add the header. This is just more correct that the filter clean up after itself and not rely on something downstream to do it later.